openassessment/xblock/ui_mixins/mfe/ora_config_serializer.py

```diff
@@ -221,3 +226,20 @@ def get_baseAssetUrl(self, block):
 
     def get_prompts(self, block):
         return [prompt["description"] for prompt in block.prompts]
+
+
+class GetBlockLearnerSubmissionDataSerializer(Serializer):
+    """
+    Main serializer for learner submission status / info
+    """
+    hasSubmitted = BooleanField(source="workflow.has_submitted")
+    hasCancelled = BooleanField(source="workflow.has_cancelled", default=False)
+    hasRecievedGrade = BooleanField(source="workflow.has_recieved_grade", default=False)
+    teamInfo = TeamInfoSerializer(source="team_info")
+    response = SerializerMethodField(source="*")
+
+    def get_response(self, data):
+        # The source data is different if we have an in-progress response vs a submitted response
+        if data['workflow']['has_submitted']:
+            return SubmissionSerializer(data['response']).data
+        return InProgressResponseSerializer(data).data
```

Comment:
This is a dupe, no? We shouldn't have this in ORA Config, since it is actually part of our submission.

https://github.com/openedx/edx-ora2/blob/0e06ac19a05a447f427595a22d78c60ddee2fd28/openassessment/xblock/ui_mixins/mfe/submission_serializers.py#L95-L109

Reply:
You're right, this is a mistake from my merge. Good catch.